homeassistant/components/husqvarna_automower/const.py

```diff
@@ -0,0 +1,33 @@
+"""The constants for the Husqvarna Automower integration."""
+from homeassistant.const import Platform
+
+# Base component constants
+NAME = "husqvarna_automower"
+DOMAIN = "husqvarna_automower"
+DOMAIN_DATA = f"{DOMAIN}_data"
+HUSQVARNA_URL = "https://developer.husqvarnagroup.cloud/login"
+OAUTH2_AUTHORIZE = "https://api.authentication.husqvarnagroup.dev/v1/oauth2/authorize"
+OAUTH2_TOKEN = "https://api.authentication.husqvarnagroup.dev/v1/oauth2/token"
+
+# Platforms
+PLATFORMS: list[Platform] = [
```

Comment:
`PLATFORMS` is only used in the init module. Please move it to that module. Only define the constants that are shared between multiple modules in the const module.